lib/grpramat.gi

```diff
@@ -335,6 +335,7 @@ function( G )
             );
     SetNiceMonomorphism(G, nice);
     SetNiceObject(G, H);
+    UseIsomorphismRelation(H, G);
```

Comment:
Perhaps it would be better to add a custom `SetNiceObject` to make sure that e.g. information about the size is automatically transferred... 

Or conversely perhaps I should *only* transfer the size here (as I know it has been computed)... And then similar questions for the code around line 275:
```gap
        # the following does not use NiceObject(G) as the only method for
        # that currently requires IsHandledByNiceMonomorphism
        SetNiceObject( G, Image( NiceMonomorphism( G ), G ) );
        return IsFinite( NiceObject( G ) );
```

@hulpke any thoughts on this?